stable/fairwinds-insights/values.yaml

```diff
@@ -479,15 +479,25 @@ ingress:
   extraPaths: {}
 
 postgresql:
-  fullnameOverride: insights-postgresql
-  # -- Set to `true` to run migrations after the upgrade
-  postMigrate: false
   image:
     registry: quay.io
     repository: fairwinds/postgres-partman
-    tag: "16.0"
-  # -- Use the ephemeral postgresql chart by default
+    tag: "17.0"
+  # -- Use the ephemeral postgresql cluster by default
   ephemeral: true
+  # -- Install CloudNativePG operator
+  operator:
+    install: true
```

Comment:
Can we remove `ephemeral.operator.install` this and only work with `postgresql.ephemeral`?